spec/requests/querying_spec.rb

```diff
@@ -85,6 +85,25 @@ def create_vms_by_name(names)
       expect(response).to have_http_status(:bad_request)
     end
 
+    it 'returns only id attributes if specified on a collection' do
+      create_vms_by_name %w(bb)
```

Comment:
Since the name isn't important here, WDYT about `vm = FactoryGirl.create(:vm)` here, then check for

```ruby
{"href" => api_vm_url(nil, vm), "id" => vm.id}
```
below?